setup.json

```diff
@@ -87,7 +87,7 @@
     "install_requires": [
         "aiida_core[atomic_tools]~=1.2",
         "packaging",
-        "qe-tools~=1.1",
+        "qe-tools~=2.0rc1",
```

Comment:
Note that this assumes we won't break backwards compatibility between the current release candidate and the "proper" release of `qe-tools`. I think that will be the case, but maybe we should pin `rc1` here just to be safe.